setup.py

```diff
@@ -1,46 +1,58 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
+from setuptools import setup
 
-with open('README.rst') as file:
-    long_description = file.read()
+# read the contents of your README file
+from os import path
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.rst'), encoding='utf-8') as f:
+    long_description = f.read()
 
-setup(
-        name = "sifter",
-        version = "0.1",
-        author = "Gary Peck",
-        author_email = "gary@realify.com",
-        url = "https://github.com/garyp/sifter",
-        license = "BSD",
-        description = "Parser/evaluator for the Sieve filtering language (RFC 5228)",
-        long_description = long_description,
-        classifiers = [
-            "Programming Language :: Python",
-            "Programming Language :: Python :: 2",
-            "License :: OSI Approved :: BSD License",
-            "Development Status :: 4 - Beta",
-            "Intended Audience :: Developers",
-            "Intended Audience :: System Administrators",
-            "Operating System :: OS Independent",
-            "Topic :: Communications :: Email :: Filters",
-            "Topic :: Software Development :: Interpreters",
-            "Topic :: Software Development :: Libraries :: Python Modules",
-            ],
-        packages = [
-            "sifter",
-            "sifter.commands",
-            "sifter.comparators",
-            "sifter.extensions",
-            "sifter.grammar",
-            "sifter.t",
-            "sifter.tests",
-            "sifter.validators",
-            ],
-        package_data = {
-            "sifter.t" : ["*.in", "*.out", "*.msg", "*.rules"],
-            },
-        install_requires=[
-            "ply"
-            ],
-        )
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
 
+setup(
+    name="sifter",
+    version="0.2.0",
+    author="Gary Peck, Manfred Kaiser",
+    author_email="gary@realify.com, manfred.kaiser@logfile.at",
+    url="https://github.com/garyp/sifter",
+    license="BSD",
+    long_description=long_description,
```

Comment:
```suggestion
    description="Parser/evaluator for the Sieve filtering language (RFC 5228)",
    long_description=long_description,
```

from what i can see the description is displayed in pypi and i think removing it will make it less clear what this tool does
![Screenshot from 2020-10-01 08-42-00](https://user-images.githubusercontent.com/2660/94776521-24bf7d00-03c2-11eb-86ad-11962429a668.png)

Reply:
Thanks, this was a mistake